packages/@o3r/design/src/core/design-token/renderers/css/design-token-updater.renderers.ts

```diff
@@ -0,0 +1,24 @@
+import type { DesignContentFileUpdater } from '../design-token.renderer.interface';
+
+const AUTO_GENERATED_START = '/* --- BEGIN THEME Auto-generated --- */';
+const AUTO_GENERATED_END = '/* --- END THEME Auto-generated --- */';
+
+const generateVars = (variables: string[], startTag: string, endTag: string, addCssScope = false) =>
+  `${addCssScope ? ':root {\n' : ''}${startTag}\n${variables.join('\n')}\n${endTag}${addCssScope ? '\n}' : ''}`;
+
+/**
+ * Retrieve a Content Updater function for CSS generator
+ * @param startTag Opening tag marking the content edition part
+ * @param endTag Closing tag marking the content edition part
+ */
+export const getCssStyleContentUpdater = (startTag = AUTO_GENERATED_START, endTag = AUTO_GENERATED_END): DesignContentFileUpdater => {
+  const regexToReplace = new RegExp(`${startTag.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')}(:?(.|[\n\r])*)${endTag.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')}`);
```

Comment:
some regexp could be shared (and a comment could be added to ease the understanding)

Reply:
Comment added and variable created but not sure where it can be shared

Reply:
`<string>.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')` is used twice on this line

Reply:
Ooooh, I did not notice....